spec/javascripts/LoginPage/LoginPageWrapper.spec.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import Enzyme, {mount} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import {SimpleLoginPage, Login3scaleForm, RequestPasswordForm} from 'LoginPage'
+
+Enzyme.configure({adapter: new Adapter()})
+
+const props = {
+  enforceSSO: false,
+  authenticationProviders: null,
+  providerAdminDashboardPath: 'provider-admin-path',
+  providerLoginPath: 'provider-login-path',
+  providerPasswordPath: 'password-path',
+  providerSessionsPath: 'sessions-path',
+  redirectUrl: 'redirect-url',
+  show3scaleLoginForm: true
+}
+
+it('should render itself with right props', () => {
```

Comment:
I'd rather test where those props are used or how they modify the component. I don't see the value in testing the the previously defined object that was passed to the component and asserting its values.